src/Controller/PetController.php

```diff
@@ -26,7 +26,7 @@ public function __construct(PetService $petService)
 
     /**
      * @Route("/new", name="new_pet")
-     * @IsGranted("ROLE_USER")
+     * @Security("is_granted('ROLE_USER') or is_granted('ROLE_ADMIN')")
```

Comment:
why is this done here? The PR says `create a user`?

Reply:
yap, different topic..sry